src/Exception/ExpectationException.php

```diff
@@ -75,6 +97,12 @@ protected function getContext()
      */
     protected function getSession()
     {
+        if (null === $this->session) {
+            throw new \LogicException(sprintf('The deprecated method %s cannot be used when passing a driver in the constructor', __METHOD__));
```

Comment:
Exception message says, that method is deprecated, but we're missing `@deprecated` in the DocBlock.